package.json

```diff
@@ -13,7 +13,10 @@
     "clean:modules": "rm -rf node_modules apps/*/node_modules packages/*/node_modules pnpm-lock.yaml",
     "clean:vitest-mjs": "find . -type f -name 'vite*.config.ts.timestamp-*-*.mjs' -ls -delete",
     "compile": "turbo compile",
-    "dev": "turbo dev",
+    "dev": "turbo dev:compile dev:app --concurrency 18",
+    "dev:app": "turbo dev:compile dev:app",
+    "dev:compile": "turbo dev:compile",
+    "dev:pack": "turbo dev:pack --concurrency 16",
```

Comment:
is there difference between difference concurrency numbers?

Reply:
concurrency limit is 10 unless set. these are precisely the necessary value. if more packages are added, or something else changes, they may need to be updated.